examples/mnist_cnn_features_level_fgsm.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""Trains a convolutional neural network on the MNIST dataset, then attacks it with the FGSM attack."""
```

Comment:
Could you please update this comment to reflect that the example attacks an inner layer?

Reply:
done